src/main/java/uk/gov/hmcts/reform/juddata/controller/WelcomeController.java

```diff
@@ -27,7 +27,7 @@
 
     private static final Logger LOG = getLogger(WelcomeController.class);
     private static final String INSTANCE_ID = UUID.randomUUID().toString();
-    private static final String MESSAGE = "Welcome to the Judicial API";
+    private static final String MESSAGE = "Welcome to the Judicial Data Load";
 
     /**
      * Root GET endpoint.
```

Comment:
i think welcome controller is not required.. can we remove this? and do smoke test with health check?

Reply:
it is just root check.else smoke test phase fails in jenkins